Frameworks/Foundation/NSURLProtocol_WinHTTP.mm

```diff
@@ -288,7 +288,8 @@ - (void)startLoading {
             // TODO #2721: If this properly supports streaming bodies instead of flattening the data up front,
             // more accurate callbacks can be done
             NSInteger contentSize = self.request.HTTPBody ? self.request.HTTPBody.length : _flattenedBodyStream.size();
-            if (contentSize > 0) {
+            if (([self.client respondsToSelector:@selector(URLProtocol:didWriteData:totalBytesWritten:totalBytesExpectedToWrite:)]) &&
```

Comment:
there are other "private" invocations in this file, are these expected to succeed?